doc/index.md

```diff
@@ -30,14 +31,14 @@ This site contains documentation for:
 - Access data from major relational databases, MongoDB, SOAP and REST APIs.
 - Incorporate model relationships and access controls for complex APIs.
 - Separable components for file storage, third-party login, and OAuth 2.0.
-" %}
+  " %}
```

Comment:
remove the empty spaces here. If it's because of auto-formatting, add a comment for prettier so that it can ignore it